eox_hooks/apps.py

```diff
@@ -38,5 +38,16 @@ class EoxHooksConfig(AppConfig):
                 'test': {'relative_path': 'settings.test'},
                 'production': {'relative_path': 'settings.production'},
             },
-        }
+        },
+        'signals_config': {
+            'lms.djangoapp': {
+                'relative_path': 'receivers',
+                'receivers': [
+                    {
+                        'receiver_func_name': 'hooks_handler',
+                        'signal_path': 'django.core.signals.request_started',
```

Comment:
I think this is a bad idea, because this will be executed in every requests and when we need to change that we hace to create a new plugin version, have you tried with settings ??'

Reply:
I was testing with the platform and left that there, sorry. I'll erase the signal_path